integration/cli-login.expect

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env expect -f
+
+#
+# SUCCESS:
+# $ dcos auth login
+# dcos-host.com's username: good
+# good@dcos-host.com's password:
+# [core.dcos_acs_token]: set
+# Login successful!
+#
+# FAILURE:
+# $ dcos auth login
+# Removed [core.dcos_acs_token]
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# dcos-host.com's username: bad
+# bad@dcos-host.com's password:
+# Authentication failed
+#
```

Comment:
Actually copied from the `.expect` file in `dcos-tests` -- ultimately I think that we should refactor out all the common logic for integration tests (there's a lot, as evidenced by the existence of `dcos-tests`) into its own library that's installable via `pip` rather than `git clone`.